validator/sawtooth_validator/execution/processor_iterator.py

```diff
@@ -32,6 +32,17 @@ class ProcessorIteratorCollection(object):
     (1 transaction processor can have multiple Handlers which will be stored
     as Processors)
 
+    Attributes:
```

Comment:
Internal attributes probably shouldn't be in the API doc string (as they are not part of the API), but rather a comment within __init__() itself as #-style comments above each attribute.